apps/design/backend/src/app.ts

```diff
@@ -47,6 +50,38 @@ import { renderBallotStyleReadinessReport } from './ballot_style_reports';
 export const BALLOT_STYLE_READINESS_REPORT_FILE_NAME =
   'ballot-style-readiness-report.pdf';
 
+enum UsState {
```

Comment:
Used in a subsequent PR when deciding which election definition generation function to use eg `createElectionDefinitionForNhTownHmpbTemplate`. Pretty unstable at the moment.